tests/playground.html

```diff
@@ -455,7 +455,7 @@
       <block type="controls_forEach"></block>
       <block type="controls_flow_statements"></block>
     </category>
-    <category name="Math" colour="230" iconclass="blocklyTreeIconCustom">
+    <category name="Math" disabled="true" colour="230" iconclass="blocklyTreeIconCustom">
```

Comment:
Might want to remove this or copy the category into a new Math_Disabled category